src/service_client/assisted_service_api.py

```diff
@@ -260,7 +260,7 @@ def download_and_save_infra_env_file(self, infra_env_id: str, file_name: str, fi
 
     def download_manifests(self, cluster_id: str, dir_path: str) -> None:
         log.info(f"Downloading manifests for cluster {cluster_id} into {dir_path}")
-        response = self.manifest.v2_list_cluster_manifests(cluster_id=cluster_id, _preload_content=False)
+        response = self.manifest.v2_list_cluster_manifests(cluster_id=cluster_id, include_system_generated="true", _preload_content=False)
```

Comment:
```suggestion
        response = self.manifest.v2_list_cluster_manifests(cluster_id=cluster_id, include_system_generated=True, _preload_content=False)
```

isn't it a boolean?

Reply:
It's a parameter used in an API call to the service, not sure how it will serialize, wanted to guarantee that it would send a string

Reply:
the boolean should be marshaled as a string in the request send to the service

Reply:
saying that because it's clear the API exposes a boolean: https://github.com/openshift/assisted-service/pull/5498/files#diff-8b1949772e223a1da6a2049ada2733fa506410975b241cf86cf44c7a8665bc62R2852

Reply:
OK, will change this to a boolean